launchpod/angular-launchpod/src/app/form-handler.service.ts

```diff
@@ -32,11 +38,27 @@ export class FormHandlerService {
     this.loginLinkSubject.next(link);
   }
 
+  /**
+   * Update the list of "my feeds" with the feeds from post request.
+   */
+  sendMyFeeds(feeds) {
+    console.log("My feeds service: " + feeds);
+    this.myFeedsSubject.next(feeds);
+  }
+
+  /**
+   * Update the newFeed boolean when a new feed is added.
+   */
+  sendNewFeed() {
```

Comment:
Nit: I know that you are stylistically keeping all the methods send_ but the name of this is confusing. It sounds like it's supposed to do what sendMyFeeds is doing but singular, so when I saw it used in create-form it took me a second to understand what was happening. 

Consider changing?

Reply:
done.